src/manifest.json

```diff
@@ -40,5 +40,7 @@
       "js/capture.js",
       "js/background.js"
     ]
-  }
+  },
+
+  "content_security_policy": "script-src 'self' 'unsafe-eval' https://aframe.io/releases/0.5.0/aframe.min.js; object-src 'self';"
```

Comment:
unrelated: can you bump the A-Frame version to 0.7.1 (can be in separate PR; I can file an issue)

Reply:
for some reasons, in the latest version I get the following error:
`THREE.Raycaster: Unsupported camera type.` from `three.js`.

Reply:
oof, is that with `0.7.0` or `0.7.1`?

Reply:
both 0.7.0 and 0.7.1

Reply:
I really would prefer to install the module locally than remotely load it, is this not possible through npm?

Reply:
possible, but I get the same error for newer versions. but will load the code via npm 👍